### PR TITLE
Vanity domain

### PR DESCRIPTION
**Pull request by joinimran**
If the vanity domain is set up at any part of Office365, CDN services will not work.

Problem: https://github.com/MicrosoftDocs/OfficeDocs-Enterprise/issues/479

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Enterprise/use-office-365-cdn-with-spo.md [hunk 1]
```diff
@@ -25,6 +25,7 @@ You can use the built-in Office 365 Content Delivery Network (CDN) to host stati
 > Restrictions for use of the Office 365 CDN:
 > + The Office 365 CDN is only available to tenants in the **Production** (worldwide) cloud. Tenants in the US Government, China and Germany clouds do not currently support the Office 365 CDN.
 > + The Office 365 CDN does not currently support tenants configured with custom or "vanity" domains. If you have added a domain to your tenant using the instructions in the topic [Add a domain to Office 365](https://docs.microsoft.com/en-us/office365/admin/setup/add-domain?view=o365-worldwide), the Office 365 CDN will return errors when you try to access content from the CDN.
+> + If vanity domain is setup for any of the Office 365 component, CDN services will not work.
```
Comment (JohanFreelancer9) [inline Enterprise/use-office-365-cdn-with-spo.md]:
```suggestion
> + If vanity domain is set up for any of the Office 365 components, the CDN services will not work.
```

**Comment by joinimran** [comment]
Copy review suggestions applied. Ready to go live.

**Comment by mypil** [comment]
@kelleyvice-msft - PR has been copy edited and is ready for final review. This is based on issue #479.

Thank you.

**Comment by kelleyvice-msft** [comment]
The proposed edit is essentially a duplicate of the paragraph directly preceding it. If the statement is intended to clarify the paragraph above, it would be much less confusing to simply amend it rather than adding a new line that seems to restate the point. Perhaps something like, "If you have added a domain to _any component of_ your tenant using the instructions..."

**Comment by mypil** [comment]
@joinimran - kindly make the necessary changes to this PR as recommended by kelleyvice-msft. 

Thank you.

**Comment by joinimran** [comment]
Made some changes to make it more clear.

**Comment by joinimran** [comment]
Suggestion applied.

**Comment by kelleyvice-msft** [comment]
Thanks, @joinimran!

**Pull request merged**